modules/rostests/apitests/ntdll/NtQueryInformationToken.c

```diff
@@ -203,7 +203,7 @@ QueryTokenOwnerTests(
     ok_ntstatus(Status, STATUS_SUCCESS);
 
     RtlConvertSidToUnicodeString(&SidString, Owner->Owner, TRUE);
-    ok_wstr(SidString.Buffer, L"S-1-5-32-544");
+    //ok_wstr(SidString.Buffer, L"S-1-5-32-544"); // broken on Windows 2003
```

Comment:
A question: Is there any difference in the meaning of your different ways to express yourself?
`broken on Windows 2003`
`broken on WHS`
`broken on Windows`

I do ike the PR very much aside that open question. I just didn't approve yet due to the Draft state.

Reply:
It all just means doesn't work on Windows. WHS == Windows 2003 x86. And since 2003 is the base, Windows == Windows 2003, otherwise I would have written the version. I can write Windows 2003 everywhere.

Reply:
Do as you please, I just wanted to understand. Resolved.